data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/KafkaHealthIndicator.java

```diff
@@ -35,7 +35,7 @@
 import java.util.stream.Collectors;
 
 /**
- * Kafka health indicator. Check that the consumer group can be accessed and is registered with the cluster,
+ * Kafka health indicator.  Check that the consumer group can be accessed and is registered with the cluster,
```

Comment:
This doesn't appear to have been updated. The check is not done on a consumer group for this service.

Reply:
Was done, something strange with Github perhaps.